lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -137,6 +137,22 @@ def initialize(model, current_user:, embed_links: false)
           }
         end
 
+        link :'custom-fields' do
```

Comment:
@oliverguenther This is in order to comply with the CSS icon class. Further, I appears to me, that is used to find the translation key, which actually seems to be applied on the frontend and not on the backend side. I could not get and backend side translation to work.